addons/ui/config.cpp

```diff
@@ -2,23 +2,43 @@
 
 class CfgPatches {
     class ADDON {
-        name = CSTRING(component);
+        name = CSTRING(Component);
         units[] = {};
         weapons[] = {};
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {"cba_common", "cba_arrays"};
         author = "$STR_CBA_Author";
-        authors[] = {"Dr Eyeball", "commy2"};
+        authors[] = {"Dr Eyeball", "commy2", "mharis001"};
         url = "$STR_CBA_URL";
         VERSION_CONFIG;
     };
 };
 
+class RscText;
+class RscProgress;
+class RscMapControl;
+class ctrlTree;
+class ctrlEdit;
+class ctrlStatic;
+class ctrlStaticTitle;
+class ctrlStaticFooter;
+class ctrlMenuStrip;
+class ctrlButtonOK;
+class ctrlButtonCancel;
+class ctrlButtonExpandAll;
+class ctrlButtonCollapseAll;
+class ctrlButtonPictureKeepAspect;
+class ctrlStaticBackground;
+class ctrlStaticBackgroundDisable;
+class ctrlStaticBackgroundDisableTiles;
```

Comment:
hate this

Reply:
The result of following BI's format for making "3DEN dialogs".